app/views/about/show.html.haml

```diff
@@ -29,7 +29,9 @@
       = render 'registration'
     - else
       .closed-registrations-message
-        - if @instance_presenter.closed_registrations_message.blank?
+        - if Rails.configuration.x.use_ldap
+          %p= t('about.ldap_registrations')
+        - elsif @instance_presenter.closed_registrations_message.blank?
```

Comment:
I don't think LDAP registrations ever need to be closed, because they're not really registrations - these people already have logins in LDAP, so it is expected that they can login on a connected Mastodon.